pkg/apis/sources/v1beta1/kafka_types.go

```diff
@@ -72,6 +72,10 @@ type KafkaSourceSpec struct {
 	// +optional
 	ConsumerGroup string `json:"consumerGroup,omitempty"`
 
+	// InitialOffset is the Initial Offset for the consumer group.
+	// +optional
+	InitialOffset *int64 `json:"initialOffset,omitempty"`
```

Comment:
We should not use sarama specific things in the API.

can we use `earliest` or `latest` as string?

/hold

Reply:
what if we want to give a specific offset ?

Reply:
Do you plan to set a single offset for all partitions?

Reply:
Or, let me ask another question, as of now, if I set 50 here, what should happen?

Reply:
sorry, I had a wrong assumption that we could also give a specific offset. 
But looks like it needs to be earliest or latest. I will change the API  to make the initial offset a string that will accept "earliest" or "latest"

Reply:
i have made the changes

Reply:
Thanks!

/unhold